gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/hive/HiveDataset.java

```diff
@@ -330,4 +335,8 @@ private boolean canCopyTable() {
     }
     return true;
   }
+
+  public Properties getProperties() {
```

Comment:
You can use lombok annotation to get rid of this method. Check "@Getter" annotation in the code base.